psydac/ddm/cart.py

```diff
@@ -1124,110 +1138,142 @@ def __init__(self, npts, pads, periods, comm, shifts, axes, exts, ranks_in_topo,
     #---------------------------------------------------------------------------
     @property
     def ndims( self ):
+        """Number of dimension."""
         return self._ndims
 
     @property
     def npts_minus( self ):
+        """Number of points in the minus side of an interface."""
         return self._npts_minus
 
     @property
     def npts_plus( self ):
+        """Number of points in the plus side of an interface."""
         return self._npts_plus
 
     @property
     def pads_minus( self ):
+        """ padding in the minus side of an interface."""
         return self._pads_minus
 
     @property
     def pads_plus( self ):
+        """ padding in the plus side of an interface."""
         return self._pads_plus
 
     @property
     def periods_minus( self ):
+        """ Periodicity in the minus side of an interface."""
         return self._periods_minus
 
     @property
     def periods_plus( self ):
+        """ Periodicity in the plus side of an interface."""
         return self._periods_plus
 
     @property
     def shifts_minus( self ):
+        """ The shift values in the minus side of an interface."""
         return self._shifts_minus
 
     @property
     def shifts_plus( self ):
+        """ The shift values in the plus side of an interface."""
         return self._shifts_plus
 
     @property
     def ext_minus( self ):
+        """ the extremity  of the boundary on the minus side of an interface."""
         return self._ext_minus
 
     @property
     def ext_plus( self ):
+        """ the extremity of the boundary on the plus side of an interface."""
         return self._ext_plus
 
     @property
     def root_rank_minus( self ):
+        """ The root rank of the communicator defined in the minus patch."""
         return self._root_rank_minus
```

Comment:
Is this the rank in the intra- or inter-communicator?

Reply:
intra

Reply:
OK! Please add "intra-" then, as this clarifies things

Reply:
done in [eab4eb9](https://github.com/pyccel/psydac/pull/213/commits/eab4eb9035ec7950199e1b03e2eaf7652bea8362)